forui/lib/src/widgets/tabs/tabs.dart

```diff
@@ -0,0 +1,167 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+import 'package:flutter_localizations/flutter_localizations.dart';
+
+import 'package:forui/forui.dart';
+
+part 'tabs_style.dart';
+
+part 'tab_controller.dart';
+
+/// An object that represents a tab entry in a group of tabs.
+class FTabEntry {
+  final String? _label;
+
+  /// A raw label.
+  final Widget? rawLabel;
+
+  /// The content of a tab.
+  final Widget content;
+
+  /// Creates a [FTabs].
+  FTabEntry({
+    required this.content,
+    String? label,
+    this.rawLabel,
+  })  : _label = label,
+        assert((label != null && rawLabel == null) || (label == null && rawLabel != null),
```

Comment:
This can be an XOR operation instead (^).

Reply:
I hope I did it right, please help me confirm that the assert statement makes sense